README.rst

```diff
@@ -609,6 +609,46 @@ Cache can be disabled via ``--using-cache`` option or config file:
         ->setUsingCache(false)
     ;
 
+Cache file can be specified via ``--cache-file`` option or config file:
+
+.. code-block:: php
+
+    <?php
+
+    return Symfony\CS\Config\Config::create()
+        ->setCacheFile(__DIR__.'/.php_cs.cache')
+    ;
+
+Using PHP CS Fixer on Travis
+----------------------------
+
+Require ``fabpot/php-cs-fixer`` as a `dev`` dependency:
+
+.. code-block:: bash
+
+    $ ./composer.phar require --dev fabpot/php-cs-fixer
+
+Create a build file to run ``php-cs-fixer`` on Travis. It's advisable to create a dedicated directory
+for PHP CS Fixer cache files and have Travis cache it between builds.
+
+.. code-block:: bash
```

Comment:
@GrahamCampbell 

Nope, needs an adjustment here: 

* https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.5/Symfony/CS/Console/Command/ReadmeCommand.php#L209

Do you want to take care of this?